etc/config/snowball.default.properties

```diff
@@ -0,0 +1,17 @@
+compilers=snowballDefault
```

Comment:
Is snowballDefault defined anywhere?

Reply:
No, I needed to use snowball.amazon.properties and link it to the local properties, it didn't detect default

Reply:
Is this a problem?

Reply:
I edited it 👍🏽

Reply:
It won't break the app or anything, but typically you'd want something like
```
compiler.snowballDefault.exe=/usr/bin/snowball
compiler.snowballDefault.name=snowball default
```

The reason being this allows snowball to show up for anyone who has snowball installed locally, and without some definition `compilers=snowballDefault` doesn't really do anything :)

Reply:
I didn't add it because I got `suspicious_path`